### DISCUSSION TITLE
请教一下，4.3.1版本文章卡片preview如何自定义meta信息

### DISCUSSION THREAD

**Discussion by tarihub** [volantis-x/hexo-theme-volantis#609] [Q&A] [answered]
如文章顶部的meta信息可以通过 _config.volantis.yml 里article -> body -> top_meta 定义，对应效果如下
![image](https://user-images.githubusercontent.com/39155974/113262740-888c1980-9303-11eb-8452-4301f7b0a515.png)

现希望在首页可以下方位置自定义相应的meta信息，如字数统计
![image](https://user-images.githubusercontent.com/39155974/113262971-cdb04b80-9303-11eb-85e0-e30d89788bc3.png)

### 自己做了的尝试
+ 在 _config.volantis.yml -> article -> preview 添加 meta 类似的字段，不过 hexo clean 后无法生效
+ 参考文档 https://volantis.js.org/v3/theme-settings/layout/ ，在 source/_data 目录下添加 layout.yml , hexo clean 后也没有明显效果
    + 之所以这样做是因为看了其他人的博客 https://www.moonback.xyz ，用的 volantis 3.0.0 可以定义首页查看文章的meta信息，然后观察了一下 3.0.0 的 .config.yml 文件有相关字段
![image](https://user-images.githubusercontent.com/39155974/113264599-b8d4b780-9305-11eb-81d9-f1e6c0b28f9a.png)

### 环境信息
```
> hexo version
INFO  Validating config
hexo: 5.4.0
hexo-cli: 4.2.0
os: Darwin 20.3.0 darwin x64
node: 12.18.1
v8: 7.8.279.23-node.38
uv: 1.38.0
zlib: 1.2.11
brotli: 1.0.7
ares: 1.16.0
modules: 72
nghttp2: 1.41.0
napi: 6
llhttp: 2.0.4
http_parser: 2.9.3
openssl: 1.1.1g
cldr: 37.0
icu: 67.1
tz: 2019c
unicode: 13.0
```
+ volantis版本: https://github.com/volantis-x/hexo-theme-volantis/tree/4.3.1
+ npm: 6.14.11

希望各位大佬能给点建议~

**Comment by xaoxuu** [comment, marked as answer]
新版本中已经去掉了这个功能

**Reply by tarihub** [reply]
好吧~ 怪不得找了这么久没找到，还以为是我姿势问题。谢啦~
